src/Estimators/OperatorEstBase.h

```diff
@@ -131,7 +136,7 @@ class OperatorEstBase
    *  And it make's datalocality fairly easy but
    *  more descriptive and safe data structures would be better
    */
-  static Data createLocalData(size_t size, DataLocality data_locality);
+  Data createLocalData(size_t size);
```

Comment:
is this function still needed? Didn't see a call.

Reply:
No it should be deleted this is an error in branch slicing.